irrexplorer/backends/common.py

```diff
@@ -48,7 +48,7 @@ async def retrieve_url_text(url: str):
     Retrieve `url` and return the text in the response.
     Raises ImporterException if the response status is not 200.
     """
-    async with aiohttp.ClientSession() as session:
+    async with aiohttp.ClientSession(headers={"User-Agent": "IRR explorer"}) as session:
```

Comment:
IRRExplorer? :)

Reply:
I actually took the name from the webpage, but I'm more than fine with that as well.

Reply:
Changed it to IRRExplorer now.